### PR TITLE
refactor: biw shortcuts

### PR DESCRIPTION
**Pull request by D4rWiNSS**
# What? 
This PR refactor some of the shorcuts of builder in world and add some missing functionality like 'Cancel' and some bug fixing

Also fixes some edge cases for selections, multi selection, and start selecting from the gizmos.

We also have refactored the input view

image

# Why? 
We have a new keybinding scheme to apply for builder in world to make it more usable

### CODE REVIEW

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs [hunk 1]
```diff
@@ -483,7 +465,8 @@ public void EnterEditMode()
 
         foreach (var groundVisual in groundVisualsGO)
         {
-            groundVisual.SetActive(false);
+            //TODO: This deactivate the ground but we may reactivate this in the future
+            ///groundVisual.SetActive(false);
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs]: In this case, temporally, we can also comment the `foreach` container, right?

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs [hunk 2]
```diff
@@ -193,6 +193,15 @@ public void RemoveLoadingObject(string entityId)
         loadingPlaceHolder.DestroyAfterAnimation();
     }
 
+    public void RemoveLoadingObjectInmediate(string entityId)
+    {
+        if (!loadingGameObjects.ContainsKey(entityId))
+            return;
+        BIWLoadingPlaceHolder loadingPlaceHolder = loadingGameObjects[entityId];
+        loadingGameObjects.Remove(entityId);
+        loadingPlaceHolder.Disspose();
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs]: Typo!
```suggestion
        loadingPlaceHolder.Dispose();
```

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/Voxels/VoxelController.cs [hunk 3]
```diff
@@ -101,20 +103,11 @@ public void SetEditObjectLikeVoxel()
         }
     }
 
-    public void SetEditionGO(GameObject _editionGO)
-    {
-        editionGO = _editionGO;
-    }
+    public void SetEditionGO(GameObject _editionGO) { editionGO = _editionGO; }
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/Voxels/VoxelController.cs]: In order to follow our code conventions...
```suggestion
    public void SetEditionGO(GameObject editionGO) { this.editionGO = editionGO; }
```

**Review by sandrade-dcl — APPROVED** [review]
Good job! Yesterday I could test the build with the new shortcuts and I think we have won a lot in terms of usability and user experience :)

Code reviewed! (I only left some little comments)

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs [hunk 4]
```diff
@@ -193,6 +193,15 @@ public void RemoveLoadingObject(string entityId)
         loadingPlaceHolder.DestroyAfterAnimation();
     }
 
+    public void RemoveLoadingObjectInmediate(string entityId)
+    {
+        if (!loadingGameObjects.ContainsKey(entityId))
+            return;
```
Comment (AjimenezDCL) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs:199]: You could use the `TryGetValue` method here:

Something like
```suggestion
        if (!loadingGameObjects.TryGetValue(entityId, out BIWLoadingPlaceHolder loadingPlaceHolder))
            return;
```

It's quite handy :). In any case this is fine, you dont have to change it

**Review by AjimenezDCL — APPROVED** [review]
Looks good!

**Pull request merged**